gatsby-config.js

```diff
@@ -36,7 +36,12 @@ const plugins = [
   'gatsby-plugin-react-helmet',
   'gatsby-plugin-sitemap',
   'gatsby-plugin-twitter',
-  'gatsby-theme-iterative-docs',
+  {
+    resolve: 'gatsby-theme-iterative-docs',
+    options: {
+      remark: false
+    }
```

Comment:
dvc.org's remark plugins are configured differently from the more basic ones from cml.dev (i.e. it has the local plugins and uses an external const for max image width), so we suppress the one from the theme and use our more specific one.